brazil.code-workspace

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is there a reason why you include this file? If not necessary, can you remove it? Thank you.